check/check.js

```diff
@@ -17,10 +17,10 @@ module.exports = (fields, locations, message) => {
     }, next);
   };
 
-  Object.keys(validator)
+  Object.keys(validatorSet)
```

Comment:
When `createCheckAPI()` mixes standard validators with custom ones, the custom ones won't be built with the flag `custom: true`.
Internally, this is used by to determine how to call the validator (or sanitizer).

A better option is to always add the standard ones from validator.js, and change `validatorSet`  to simply `customVaildators`, then extend them like it's done a few lines below with `isArray`/`isString`/`exists`.